recipes/agg/all/conanfile.py

```diff
@@ -0,0 +1,140 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import copy, get, rmdir
+from conan.errors import ConanInvalidConfiguration
+
+import os
+
+required_conan_version = ">=1.52.0"
+
+class AggConan(ConanFile):
+    name = 'agg'
+    description = 'AGG Anti-Grain Geometry Library'
+    topics = ('graphics')
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = 'https://github.com/aggeom'
```

Comment:
It appears this is a fork of the main repo which is still very active https://github.com/ghaerr/agg-2.6

We generally do not accept these, is there a reason you did not use the original or contribute to it? I saw they made improvements to their CMake just 2 days ago

Reply:
Of course, my pr may look strange and I understand your position. Let me explain my position. Firstly, I don't see any commits since Jul 21, 2022. Secondly, it contains only 19 commits from May 6, 2019. Others commits do not relay to current repo and current owner. Thirdly, 2.6.0 was released in 2018 and no releases were published in current repo. You said it active, maybe. I'm not sure how to classify this is active and this is not. You can look at pull requests https://github.com/ghaerr/agg-2.6/pulls. There are 2 prs, and they related to cmake improvements, they were not reviewed for years.  If you are strongly against using fork, I will create another third pull request. But I'd be willing to bet that no one will review it or accept for years.

Reply:
In such a case I would recommend using our `author-` prefix so that it's more clearly evident that this is a specific variant of a project :)

this way people can benifit from the project while not getting mixed up :)

Reply:
https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#what-is-the-policy-on-recipe-name-collisions

🤔  I need to write better docs for that....

Reply:
@prince-chrismc thank you, I applied your suggestion but looks like cla bot is broken.